MIDAS/src/systems.cpp

```diff
@@ -76,8 +76,8 @@ DECLARE_THREAD(gas, RocketSystems* arg) {
 
 DECLARE_THREAD(voltage, RocketSystems* arg) {
     while (true) {
-        THREAD_SLEEP(16);
-        //Serial.println("VOLT");
+        arg->rocket_state.voltage.update(arg->sensors.voltage.read())
```

Comment:
semicolon needed at the end

Reply:
what python (and not being able to compile) does to a mf

Reply:
wait why can't you compile

Reply:
sent the log on slack